### ISSUE TITLE
Navigation Buggy after Updating Livewire to 2.6.1

### ISSUE THREAD

**Issue by bomshteyn** [rappasoft/laravel-livewire-tables#457]
The URL jumps back to previous page about a second after navigating between livewire pages while showing the correct content.

Downgrading to livewire 2.5.5 solves the issue. 

I narrowed it down to pages that are using Livewire Tables Package so i am guessing it's somehow related to it.

related discussion: https://github.com/livewire/livewire/discussions/3725

**Comment by cswni** [comment]
Same situation.
Downgrading to livewire 2.5.5 solves the issue.

**Comment by rappasoft** [comment]
[Looks like they fixed it](https://github.com/livewire/livewire/discussions/3725#discussioncomment-1332424)

**Comment by pktharindu** [comment]
I can confirm this is now fixed with Livewire v2.6.2.

**Comment by rizkhal** [comment]
Hello, i think i have the similliar problem but only on production, see [here](https://github.com/livewire/livewire/discussions/3725#discussioncomment-1422629) for detail..

The version i use
```
  "livewire/livewire": "^2.5.5",
  "rappasoft/laravel-livewire-tables": "1.15.0",
```
